parquet-column/pom.xml

```diff
@@ -76,6 +76,18 @@
       <version>${slf4j.version}</version>
     </dependency>
 
+    <dependency>
+      <groupId>org.locationtech.jts</groupId>
+      <artifactId>jts-core</artifactId>
+      <version>${jts.version}</version>
+    </dependency>
+
+    <dependency>
+      <groupId>net.sf.geographiclib</groupId>
+      <artifactId>GeographicLib-Java</artifactId>
```

Comment:
@jiayuasu Do you have any concern about its license?

Reply:
@wgtmac I have discussed with @zhangfengcdt . Currently, we don't have an easy way to correctly compute the polygonal covering of spherical edge. So in this implementation, we will throw a `not-implemented` exception for the covering statistics, when the spherical edge is specified.

We will update the PR to revert the GeographicLib dependency early next week.

Reply:
@wgtmac @jiayuasu I have updated the PR to reflect the discussion and decision here.